### ISSUE TITLE
Option to use system fonts

### ISSUE THREAD

**Issue by naiyerasif** [Mosc/Glider#104]
Please provide an option to use System fonts in v2 instead of default app fonts. For me, the system font is thicker and much more legible.

Alternatively, include more legible fonts, such as [Inter](https://github.com/rsms/inter), [IBM Plex Sans](https://github.com/IBM/plex), etc. This might bloat the package unless you plan to use [downloadable fonts](https://developer.android.com/develop/ui/views/text-and-emoji/downloadable-fonts).

**Comment by Mosc** [comment]
Duplicate of #49 and #94. I'd love to add support for using the system font, but that's unfortunately blocked by the Flutter framework. I'm considering adding support for alternative fonts.
